plugins/modules/copr.py

```diff
@@ -52,6 +52,16 @@
               for example V(epel-7-x86_64). Default chroot is determined by the operating system,
               version of the operating system, and architecture on which the module is run.
         type: str
+    includepkgs:
+        description: List of packages to include.
+        required: false
+        type: str
```

Comment:
This is a string, not a list (of strings). If you write "list" in the description, Ansible users expect that this has `type=list`.

How about making this a list:
```suggestion
        type: list
        elements: str
```
and combining that back to a string in the code?

Reply:
I can update that the change in description and  would make this changes in code 
```
module_args = dict(
    name=dict(type="str", required=True),
    state=dict(type="str", choices=["enabled", "disabled", "absent"], default="enabled"),
    chroot=dict(type="str"),
    includepkgs=dict(type="list", elements="str", required=False),
    excludepkgs=dict(type="list", elements="str", required=False),
)
```
Also there would be slight change in logic , for user to input list then the list would have to be converted into string seperated  by commas
```
if self.ansible_module.params.["includepkgs"]:
    includepkgs_value = ','.join(self.ansible_module.params['includepkgs'])
    repo_content = repo_content.rstrip('\n') + '\nincludepkgs={0}\n'.format(includepkgs_value)
```
is this what you are asking @felixfontein

Reply:
Yes, that looks good to me. If a user provides a comma-separated string instead of a list to the module, Ansible will automatically convert it to a list (splitting by comma) - so in this case nothing changes. *But* if a user provides a real list, it will also be handled correctly.

Reply:
lets hope it works 
Pushing with updated changes
🤞 😎

Reply:
Did you test it with the changes?

Reply:
Yes , tested it . No issues working fine